FacesofLondon/app/src/main/java/com/novoda/wallpaper/droidcon/TimeOfDayCalculator.java

```diff
@@ -1,12 +1,18 @@
 package com.novoda.wallpaper.droidcon;
 
-import java.util.Calendar;
-
 class TimeOfDayCalculator {
 
+    private static int flag = 0;
+
     TimeOfDay currentTimeOfDay() {
-        Calendar calendar = Calendar.getInstance();
-        int hourOfDay = calendar.get(Calendar.HOUR_OF_DAY);
-        return TimeOfDay.forHourOfDay(hourOfDay);
+        TimeOfDay[] values = TimeOfDay.values();
+        flag++;
+        if (flag >= values.length) {
+            flag = 0;
+        }
```

Comment:
This was actually some code I used for easy debugging and wasn't meant to be in the PR 😅 
Nice way of writing it though. Never occurred to me like that